internal/cacerts/cacerts.go

```diff
@@ -0,0 +1,14 @@
+// Package cacerts provides an embedded CA root certificates bundle.
+package cacerts
+
+//go:generate curl -sfL https://curl.se/ca/cacert.pem -o ca-certificates.crt
```

Comment:
Do we want `go generate ./...` running CI so this is always up to date?

Reply:
I don't think so? I'd love some advice from an internet friend on that one.

Reply:
We don't want CI to generate commits - but CI probably should validate that there aren't any diffs when re-running go:generate in general.

Reply:
on second thoughts - we might want this update to be separate from `go:generate` - adding an online-network requirement to a build step is something I don't love.

Reply:
I changed this to be a comment rather than a `go:generate`